index.js

```diff
@@ -176,6 +179,22 @@ class mochaPlugin {
               mocha.reporter(reporter, reporterOptions);
             }
 
+            const compilers = myModule.options['compilers'];
+            if (compilers !== undefined) {
+              const extensions = ['js'];
+              myModule.options['compilers'].split(' ').filter(function(e){return e!==''}).forEach(function (c) {
+                const idx = c.indexOf(':');
```

Comment:
I would rather split the c with the first occurrence of `:` and use those parts, perhaps with reduce. 
```
c.split(/:(.+)/)
  .reduce(
```